.drone/travis_install.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -ex
+
+
+section "install.base.requirements"
+pip install --upgrade pip
+hash -d pip  # find upgraded pip
+pip install --retries 3 -q requests six python-dateutil nose nose-exclude mock
```

Comment:
None of these may be necessary any more.  Perhaps requests still used now before tornado refactor.